script/build.js

```diff
@@ -422,7 +422,12 @@ if (options.watch) {
     historyApiFallback: {
       rewrites: [
         { from: '^/track-claims(.*)', to: '/track-claims/' },
-        { from: '^/education/apply-for-education-benefits/application(.*)', to: '/education/apply-for-education-benefits/application/' },
+        { from: '^/education/apply-for-education-benefits/application/1990(\/.*)?$', to: '/education/apply-for-education-benefits/application/1990/' }, // eslint-disable-line no-useless-escape
```

Comment:
Thanks to @cvalarida for finding an elegant solution for this.

Reply:
The 1990 redirect adds several `/`'s before navigating to the introduction page. I'm not sure why this is happening only here (for the 1990). Any thoughts on this @jbalboni @cvalarida?